app/services/achiever/course/template.rb

```diff
@@ -68,6 +68,7 @@ def self.all
     templates.map { |course| Achiever::Course::Template.from_resource(course, activities) }
   end
 
+  # @return [Achiever::Course::Template] !!raises ActiveRecord::RecordNotFound!! if none
   def self.find_by_activity_code(activity_code)
     templates = all
     template = templates.find { |val| val.activity_code == activity_code.upcase }
```

Comment:
Can we add a similar message to this raise?

```
raise ActiveRecord::RecordNotFound, "Could not find template #{activity_code} in Smart Connector" unless template
```

Reply:
You mean to logs and Sentry? I guess so but we've got on for 4 years without it.

Reply:
I mean to just include it as a message in the error that is already being thrown so we can get some more useful information when we do see the exception. Logging to sentry could also be helpful but generally, that is handled where the exception is caught